src/content/post/a-nerdy-editorial-experiment.md

```diff
@@ -0,0 +1,45 @@
++++
+date = "2016-04-08T11:02:30Z"
```

Comment:
change date to `2016-04-05` to follow a right timeline (https://github.com/sparkfabrik/sparkfabrik.github.com/pull/23)